hubblestack/__init__.py

```diff
@@ -1,3 +1,5 @@
-__version__ = '4.0.0'
+# coding: utf-8
 
-__buildinfo__ = {'branch': 'BRANCH_NOT_SET', 'last_commit': 'COMMIT_NOT_SET'}
\ No newline at end of file
+from hubblestack.version import __version__
```

Comment:
No, it has to be in the hubblestack namespace so it can be imported. The other logical place is in __init__ where it used to be, but several of the old salt-esque packages expect to find it in .version ...